python/tvm/micro/testing.py

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Defines testing methods used with microTVM."""
+
+import pathlib
+import json
+from typing import Union
+
+
+def _check_tune_log(log_path: Union[pathlib.Path, str]):
+    """Reads tune log and check each result"""
+    results = []
+    with open(log_path, "r") as f:
+        line = f.readline()
+        while line:
+            results.append(json.loads(line))
+            line = f.readline()
+
+    for item in results:
+        tune_result = item["result"]
+        assert tune_result[0][0] < 1000000000.0
```

Comment:
These two loops seems like can get merge into one.

Reply:
Thanks for the review.
If this is the only change, I will fix it in the follow on PR: https://github.com/apache/tvm/pull/9026

Reply:
fixed it.